Tutorials/whatiswot/15-Protocol_Level/README.md

```diff
@@ -0,0 +1,15 @@
+# Video 15: IoT Protocols in Thing Description
+
+## Table of Contents
+
+- Introduction
+- Forms Structure
+    - WoT Operations
+    - Protocol and URI
+    - Content Type
+    - Protocol Specific Vocabulary
+- Often used protocols
```

Comment:
```suggestion
- Common protocols
```

Reply:
@relu91 that can be understood as protocols that are common to all applications though

Reply:
```suggestion
- Typical protocols
```